### ISSUE TITLE
Shear envelopes discontinuous at overhangs

### ISSUE THREAD

**Issue by Lomarandil** [buddyd16/theToolbox#7]
An inconsequential issue, but the shear curves don't connect at overhangs
![image](https://user-images.githubusercontent.com/45415987/167246502-9409d337-aa5a-4f57-a9e7-3742c9f95c89.png)

**Comment by buddyd16** [comment]
yeah this is a consequence of the cantilevers being different data sets from the main span. I'll think on this one to see if I can work out a good way to close the diagrams.
